src/lib/frontend/components/ProductCard.svelte

```diff
@@ -57,46 +55,50 @@
-                     in:scale={{ duration: 500, opacity: 0.5, start: 0.5, easing: quintOut }}>
-                    {name}
-                </div>
-            {/if}
-            <div class={`${expanded ? 'w-4/12' : 'w-full'} flex flex-col`}>
-                {#if (percentage && expanded)}
-                    <div class="w-full flex-col items-center justify-center text-xl font-semibold">
-                        ${getPriceFirstPart(salePrice)}
-                        <sup>{getPriceSecondPart(salePrice)}</sup>
-                        {#if (isSoldByQuantities)}
-                            <span class="text-m block text-right pr-1"><sub>/ {getAbbreviatedUnit(unit)}</sub></span>
-                        {/if}
-                    </div>
-                {/if}
-                <div
-                        class={`w-full items-center justify-center ${expanded ? "flex-col" : "flex"} ${expanded && percentage ? "line-through text-lg font-light" : "text-xl font-semibold"}`}>
-                    <span class="block">${getPriceFirstPart(regularPrice)}<sup>{getPriceSecondPart(regularPrice)}</sup></span>
+
+    <div class="w-full flex-col justify-start items-start gap-2.5 flex">
+        <div class="w-full justify-between items-center flex">
+            <div class="text-stone-900 text-xs font-normal pl-1.5 overflow-hidden max-h-[35px]">{name}</div>
+
+            {#if (!percentage)}
+                <div class="text-center text-teal-950 text-15 font-bold flex ml-3">
+                    CAD${getPriceFirstPart(regularPrice)}.{getPriceSecondPart(regularPrice)}
                     {#if (isSoldByQuantities)}
-                        <span class="{expanded ? 'text-m block text-right pr-1' : 'text-sm'}"><sub>/ {getAbbreviatedUnit(unit)}</sub></span>
+                        <span>/{getAbbreviatedUnit(unit)}</span>
                     {/if}
                 </div>
-            </div>
+            {:else}
+                <div class="flex-col justify-start items-end gap-0.5 inline-flex">
+                    <div class="flex items-end leading-normal flex-col">
+                        <span class="text-center text-teal-950 text-[8px] font-bold line-through">
+                            CAD${getPriceFirstPart(regularPrice)}.{getPriceSecondPart(regularPrice)}
+                        </span>
+                        <div class="text-center text-red600 text-15 font-bold flex flex-row ml-3">
+                            CAD${getPriceFirstPart(salePrice)}.{getPriceSecondPart(salePrice)}
+                            {#if (isSoldByQuantities)}
+                                <span>/{getAbbreviatedUnit(unit)}</span>
+                            {/if}
+                        </div>
+                    </div>
+                </div>
+            {/if}
         </div>
     </div>
 </div>
 
 <style lang="css">
-    .product-image {
-        aspect-ratio: 1;
+    .discount-bg-low {
```

Comment:
Can we achieve this using tailwin? Because we might have to illustrate discount in other places than in the ProductCard.